Abhi-AnimeX/app/src/main/java/com/abhijith/animex/domain/models/AnimeItem.kt

```diff
@@ -0,0 +1,41 @@
+package com.abhijith.animex.domain.models
+
+import com.abhijith.animex.data.network.model.AnimeDto
+
+data class AnimeItem(
+    val title: String,
+    val source: String,
+    val yearAndJapaneseName: String,
+    val rating: String,
+    val rank: String,
+    val score: String,
+    val popularity: String,
+    val synopsis: String,
+    val genres: List<String>,
+    val imageUrl: String,
+    val youtubeId: String
+)
+
+
+fun AnimeDto.toDomain(): AnimeItem {
```

Comment:
fun can be part of data class like, `data class ABC() {...}`

Reply:
im confused to which class do i add this under... is it under the DTO (in data layer) or the domain class?